### ISSUE TITLE
Enhanced servers tab

### ISSUE THREAD

**Issue by RyanTheAllmighty** [ATLauncher/ATLauncher#421]
Currently I see users trying to use the Servers tab to manage their servers more and expecting certain things which aren't there.

While the `LaunchServer` scripts should stay the same and not be modified, the launcher itself should handle the following:

- Have a `Settings` button mimicking the settings of Instances
- When launching, do so manually (not using the scripts) and use a combination of the launchers settings mixed with the Servers settings (as above) again like Instances do
- Switch the buttons to use the new DropDownButtons used in the Get Help button on Instances to save space
- Route logs to the launchers console to make support easier
- Add in buttons for `Add Mods` and `Edit Mods`
- Add button for `Copy Mods` which will copy the mods from an instance of the same Minecraft version to the server

**Comment by cweimann** [comment]
I see a check by "Add in buttons for Add Mods and Edit Mods" and what appears to be a related commit but it doesn't seem to be in my launcher.  Is this not part of a release yet?  I'm running  ATLauncher Version: 3.4.38.2 [4928746f0fdbc639c44280774ddd15e905826e76]
Thanks in advance.

**Comment by RyanTheAllmighty** [comment]
> I see a check by "Add in buttons for Add Mods and Edit Mods" and what appears to be a related commit but it doesn't seem to be in my launcher. Is this not part of a release yet? I'm running ATLauncher Version: 3.4.38.2 [4928746f0fdbc639c44280774ddd15e905826e76] Thanks in advance.

Correct, it's slated for the next release which I need to get back to finishing, hopefully this weekend I can find the time
